extensions/skins/conversation_v1/Conversation.js

```diff
@@ -208,11 +208,22 @@ oppia.directive('conversationSkin', [function() {
       $scope.PANEL_TUTOR = 'tutor';
       $scope.PANEL_SUPPLEMENTAL = 'supplemental';
 
+      $scope.helpHtml = null;
+      $scope.continueToNextCardHtml = null;
```

Comment:
Done -- went with $scope.helpCardHtml and $scope.helpCardHasContinueButton. I originally did this but felt that there were too many conditionals; on reconsideration I think it's worth trying to combine them as you suggest, though (especially as they're now located in the same place).